### ISSUE TITLE
stopped working after linux update (energy_now --> charge_now)

### ISSUE THREAD

**Issue by danmilon** [NuckChorris/assault#14]
```
$ uname -a
```
```
Linux danmilon-laptop 3.18.2-2-ARCH #1 SMP PREEMPT Fri Jan 9 07:37:51 CET 2015 x86_64 GNU/Linux
```

As you  can see, `energy_now` has been renamed to `charge_now`.

```
$ find /sys/class/power_supply/BAT0/
```
```
/sys/class/power_supply/BAT0/
/sys/class/power_supply/BAT0/temp
/sys/class/power_supply/BAT0/type
/sys/class/power_supply/BAT0/alarm
/sys/class/power_supply/BAT0/power
/sys/class/power_supply/BAT0/power/control
/sys/class/power_supply/BAT0/power/wakeup_prevent_sleep_time_ms
/sys/class/power_supply/BAT0/power/async
/sys/class/power_supply/BAT0/power/wakeup_abort_count
/sys/class/power_supply/BAT0/power/wakeup_active
/sys/class/power_supply/BAT0/power/wakeup_total_time_ms
/sys/class/power_supply/BAT0/power/wakeup_active_count
/sys/class/power_supply/BAT0/power/runtime_enabled
/sys/class/power_supply/BAT0/power/runtime_active_kids
/sys/class/power_supply/BAT0/power/runtime_active_time
/sys/class/power_supply/BAT0/power/wakeup_max_time_ms
/sys/class/power_supply/BAT0/power/wakeup_count
/sys/class/power_supply/BAT0/power/wakeup_last_time_ms
/sys/class/power_supply/BAT0/power/wakeup
/sys/class/power_supply/BAT0/power/autosuspend_delay_ms
/sys/class/power_supply/BAT0/power/runtime_status
/sys/class/power_supply/BAT0/power/runtime_usage
/sys/class/power_supply/BAT0/power/wakeup_expire_count
/sys/class/power_supply/BAT0/power/runtime_suspended_time
/sys/class/power_supply/BAT0/charge_full_design
/sys/class/power_supply/BAT0/current_avg
/sys/class/power_supply/BAT0/current_now
/sys/class/power_supply/BAT0/charge_now
/sys/class/power_supply/BAT0/charge_full
/sys/class/power_supply/BAT0/device
/sys/class/power_supply/BAT0/capacity
/sys/class/power_supply/BAT0/subsystem
/sys/class/power_supply/BAT0/status
/sys/class/power_supply/BAT0/voltage_now
/sys/class/power_supply/BAT0/uevent
/sys/class/power_supply/BAT0/model_name
/sys/class/power_supply/BAT0/manufacturer
/sys/class/power_supply/BAT0/technology
/sys/class/power_supply/BAT0/cycle_count
/sys/class/power_supply/BAT0/voltage_min_design
/sys/class/power_supply/BAT0/present
```

**Comment by NuckChorris** [comment]
Assuming AwesomeWM, this should be fixed by #13 now :+1:

**Comment by danmilon** [comment]
Right! Thank you.
